### ISSUE TITLE
find outdated CAS registry numbers

### ISSUE THREAD

**Issue by egonw** [bridgedb/tiwid#25]
See https://duckduckgo.com/?q=site%3Acommonchemistry.cas.org+%22Deleted+or+Replaced+CAS+Registry+Numbers%22&t=brave&ia=web

**Comment by cthoyt** [comment]
i've written a client for the CAS API which does return the replaced/deprecated numbers for each: https://github.com/cthoyt/chemical-abstracts-service-client

I haven't figured out what's the rate limiting strategy for the API yet. And of course, you have to know which CAS numbers to look for

**Comment by egonw** [comment]
> i've written a client for the CAS API which does return the replaced/deprecated numbers for each:

Yes, but because of copyright law, we can likely not use this for this CCZero knowledge base.

Hence the "done manually".
